### ISSUE TITLE
Switch to StandardJS rule to enforce single quotes

### ISSUE THREAD

**Issue by dannobytes** [polleverywhere/js-config#11]
@nguyenj @steel @adam-h 

https://github.com/polleverywhere/js-config/blob/18d78637b8db69c593d8bc7b074b28b62cdf2973/eslint-config/index.js#L14

This is currently forcing us to use "double" quotes, but if we should switch to single-quotes. This is an auto-fixable rule, I believe. Worked fine in web_presenter.

Can we remove this rule? Or perhaps switch to "warn" as an intermediary step?
e.g.
```
"quotes": ["warn", "single", { "avoidEscape": true, "allowTemplateLiterals": true }], 
```

**Comment by adam-h** [comment]
Do you know if the rest of the exceptions we have can be auto-fixed? If so, maybe we do it all at once and release as v2

**Comment by dannobytes** [comment]
Yep, that sounds good to me. I will spend a minute today checking some
other repos and see if any complications come up.

**Comment by dannobytes** [comment]
I just tested toolkit switching over to full `es-config-standard` with no problems. Tests all pass. So I'm pushing up a PR for that now.

I imagine Core is now up to date to ES6 standards too and should probably switch over.

For what other repos (besides rails) do we need to continue maintaining our own eslint config exceptions for?

**Comment by adam-h** [comment]
Sounds good. I've updated mobile and here's what I can think of as what needs to be done:

 - [ ] Core [PR](https://github.com/polleverywhere/core/pull/102)
 - [ ] Toolkit [PR](https://github.com/polleverywhere/toolkit/pull/266)
 - [x] Mobile
 - [ ] Viz
 - [ ] Web Presenter
 - [ ] Rails

**Comment by dannobytes** [comment]
Thanks for listing those.

Ugh, I know we're already underway here. But I had a thought and now wonder if the better way to streamline and propagate the same linter everywhere is to have each repo install and use @polleverywhere/eslint-config instead of installing eslint-config-standard independently. Then we can just update a single eslintrc file in this repo, while also updating the package.json in this repo to include all the necessary dependencies of eslint-config-standard so each repo doesn't need to manually install all of them.

For any other repos currently using @pollev.../esling-config, they can just stay at the old version, while we publish 2.0.0 with the new eslintrc file with full eslint rules.

Thoughts @nguyenj @adam-h ?

**Comment by nguyenj** [comment]
Whoops, just realized there's a list in this issue. Moving it to the top so it's more visible.
